python/requirements/CI/wheel-build.txt

```diff
@@ -0,0 +1,6 @@
+attrs
+hypothesis
+humanize
+nose
+numpy
```

Comment:
Probably need to pin this at something very old. There's an issue about this (on my phone)

Reply:
#70